### ISSUE TITLE
Add `generate component` CLI command

### ISSUE THREAD

**Issue by timriley** [hanami/cli#145]
Example:

`hanami generate component some.deep.nested.my_class`

```ruby
module MyApp
  module Some
    module Deep
      module Nested
        class MyClass
        end
      end
    end
  end
end
```

**Comment by krzykamil** [comment]
I think this one can be moved to `FOR REVIEW` section

**Comment by timriley** [comment]
Thanks @krzykamil, I promise I'll review your PR on this soon!

**Comment by timriley** [comment]
Fixed by #130
